### PR TITLE
delly 0.8.3

### PR DESCRIPTION
**Pull request by tseemann**
- [ ] Have you followed the [guidelines for contributing](https://github.com/brewsci/homebrew-bio/blob/master/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/brewsci/homebrew-bio/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source FORMULA`, where `FORMULA` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict FORMULA` (after doing `brew install FORMULA`)?

-----

### CODE REVIEW

File: Formula/delly.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class Delly < Formula
+  # cite Rausch_2012: "https://doi.org/10.1093/bioinformatics/bts378"
+  desc "Structural variant discovery by paired-end and split-read analysis"
+  homepage "https://github.com/tobiasrausch/delly"
+  url "https://github.com/tobiasrausch/delly.git",
+      :tag      => "v0.8.3",
+      :revision => "56ab2900da269568dc327045913ae77ff3919df1"
+
+
```
Comment (sjackman) [inline Formula/delly.rb]: 👇Is this magic, or what! Thank you, @jonchang!

Comment (tseemann) [reply Formula/delly.rb]: Very nice !

**Comment by tseemann** [comment]
```
g++ -isystem /private/tmp/delly-20200312-5040-tb3g73/src/htslib/ -pedantic -W -Wall -Wno-unknown-pragmas -D__STDC_LIMIT_MACROS -fno-strict-aliasing -fpermissive -fopenmp -DOPENMP -O3 -fno-tree-vectorize -DNDEBUG src/delly.cpp -o src/delly -L/private/tmp/delly-20200312-5040-tb3g73/src/htslib/ -L/private/tmp/delly-20200312-5040-tb3g73/src/htslib//lib -lboost_iostreams -lboost_filesystem -lboost_system -lboost_program_options -lboost_date_time  -static -static-libgcc -pthread -lhts -lz -llzma -lbz2
In file included from /usr/local/include/boost/config/header_deprecated.hpp:18,
                 from /usr/local/include/boost/progress.hpp:22,
                 from src/delly.h:26,
                 from src/delly.cpp:17:
/usr/local/include/boost/progress.hpp:23:1: note: #pragma message: This header is deprecated. Use the facilities in <boost/timer/timer.hpp> or <boost/timer/progress_display.hpp> instead.
   23 | BOOST_HEADER_DEPRECATED( "the facilities in <boost/timer/timer.hpp> or <boost/timer/progress_display.hpp>" )
      | ^~~~~~~~~~~~~~~~~~~~~~~
/usr/local/include/boost/timer.hpp:21:1: note: #pragma message: This header is deprecated. Use the facilities in <boost/timer/timer.hpp> instead.
   21 | BOOST_HEADER_DEPRECATED( "the facilities in <boost/timer/timer.hpp>" )
      | ^~~~~~~~~~~~~~~~~~~~~~~
ld: library not found for -lcrt0.o
```

**Comment by sjackman** [comment]
`-static -static-libgcc` Try removing these options from the `Makefile`.

**Comment by tseemann** [comment]
Thanks @jonchang  - i feel this will become an exemplar for me on getting Macos + boost + openmp all working together with clang!

**Pull request merged**